packages/fast-components/src/switch/switch.styles.ts

```diff
@@ -63,8 +64,8 @@ export const SwitchStyles = css`
 
     .checked-indicator {
         position: absolute;
-        height: calc((${heightNumber} - (var(--design-unit) * 2)) * 1px);
-        width: calc((${heightNumber} - (var(--design-unit) * 2)) * 1px);
+        height: calc((${heightNumber} - (var(--design-unit) * 5.5)) * 1px);
```

Comment:
I'm not going to block on this since I'm behind in working on the spec for handling density, but I'd like to see us calculate sizes by something that provides some declarative way to understand what's going on and avoid magic number or fractions that mostly happen to produce the desired value. I'll look into this more holistically though.